### PR TITLE
Get rid of fields "phone" and "name"

### PR DESCRIPTION
**Pull request by jerben**
The user management app is mainly concerned with bringing in an
identifiable user to deal with authentication. If stuff like addresses
are needed, they have to be provided by the domain app.

### CODE REVIEW

File: sihl_users/lib/command.ml [hunk 1]
```diff
@@ -8,8 +8,7 @@ let fn request args =
   | [ "createadmin"; email; password ] -> (
       let* result =
         try_to_run (fun () ->
-            Service.User.create_admin request ~email ~password ~username:"admin"
-              ~name:"admin")
+            Service.User.create_admin request ~email ~password ~username:None)
```
Comment (aronerben) [inline sihl_users/lib/command.ml:11]: @jerben Why set this to `None`?

Comment (aronerben) [reply sihl_users/lib/command.ml:11]: Why make it optional?

Comment (jerben) [reply sihl_users/lib/command.ml:11]: @aronerben Most of the services don't force me to have a username, but email seems to be mandatory pretty much everywhere. Also we can later on use OAuth to pull information from Github, Google, Facebook etc. while most of the time we won't get a username.

Comment (aronerben) [reply sihl_users/lib/command.ml:11]: Ok, makes sense, ty!

**Pull request merged**